lua/telescope/builtin/__files.lua

```diff
@@ -538,13 +540,23 @@ files.current_buffer_fuzzy_find = function(opts)
       previewer = conf.grep_previewer(opts),
       attach_mappings = function()
         action_set.select:enhance {
```

Comment:
```suggestion
        actions.select_default:replace(function(prompt_bufnr)
          local selection = action_state.get_selected_entry()
          if not selection then
            utils.__warn_no_selection "builtin.current_buffer_fuzzy_find"
            return
          end
          local current_picker = action_state.get_current_picker(prompt_bufnr)
          local searched_for = require("telescope.actions.state").get_current_line()
          local highlighted = current_picker.sorter:highlighter(searched_for, selection.ordinal)
          local column = highlighted[1] or 0
          actions.close(prompt_bufnr)

          vim.schedule(function()
            vim.api.nvim_win_set_cursor(0, { selection.lnum, column })
          end)
        end)
```
Minor touch up so we can do this all is one rather than split across `pre` and `post` operations.